dispatch/tests/test_api_pages.py

```diff
@@ -0,0 +1,239 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from dispatch.tests.cases import DispatchAPITestCase
+
+from dispatch.apps.content.models import Page, Person
```

Comment:
The `Person` model is used to store information about authors and photographers, as well any users in the system. Not every `Person` is a user, but those that are have a corresponding `User` model that can be accessed via `person.user`